dev/nuxt/pages/index.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <h2>vue-tinybox demo: Nuxt.js</h2>
+    <p>Click on any image below to open <code>Tinybox</code></p>
+
+    <client-only>
+      <Tinybox
+        v-model="index"
+        :images="images"
+        no-thumbs
+      />
+    </client-only>
+    <img
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Elements in iteration expect to have 'v-bind:key' directives.](https://app.codacy.com/manual/NickKaramoff/vue-tinybox/pullRequest?prid=4749911)